test/polytopes.jl

```diff
@@ -139,13 +139,12 @@
     h = Hexahedron(P3[(0,0,0),(1,0,0),(1,1,0),(0,1,0),
                       (0,0,1),(1,0,1),(1,1,1),(0,1,1)])
     @test volume(h) ≈ T(1*1*1)
-    h = Hexahedron(P3[(0,0,0),(2,0,0),(2,2,0),(0,2,0),
+    h = Hexahedron(P3[(0,0,0),(1,0,0),(1,1,0),(0,1,0),
                       (0,0,2),(2,0,2),(2,2,2),(0,2,2)])
-    @test volume(h) ≈ T(2*2*2)
+    @test volume(h) ≈ T((1 + 4 + sqrt(1*4))*2/3)
```

Comment:
Need to explain where this formula is coming from so that future readers can quickly understand the test.